plugins/by-name/hmts/default.nix

```diff
@@ -24,9 +23,9 @@ in
   };
 
   config = mkIf cfg.enable {
-    warnings = optional (!config.plugins.treesitter.enable) [
-      "Nixvim: hmts needs treesitter to function as intended"
-    ];
+    warnings = optional (
+      !config.plugins.treesitter.enable
+    ) "Nixvim: hmts needs treesitter to function as intended";
```

Comment:
We could achieve the same thing with a smaller diff by doing `optional` -> `optionals`

Reply:
True, just thought you preferred the optional for single lists :P

Reply:
No real preference. `optional` is often neater, however nixfmt seems to not be a massive fan looking at this :grin:

Reply:
They definitely aren't... that's why I rarely used it because it's often discouraged in nixpkgs.